### DISCUSSION TITLE
Can't debug Storybook components of a Next.js project

### DISCUSSION THREAD

**Discussion by kemgn** [storybookjs/storybook#26873] [Help]
### Summary

First I create a Next.js App: `npx create-next-app@latest`
Then I install Storybook: `npx storybook@latest init`

When I put debugger on the boilerplate Button component, it hits the breakpoint but the browser throws this error: 

`Could not load content for webpack://ims-component-library/src/stories/Components/Atoms/Button/Button.tsx (Fetch through target failed: Unsupported URL scheme; Fallback: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME)`

![image](https://github.com/storybookjs/storybook/assets/76118092/aaf1cfd7-ee2b-4ed9-b220-f7a2e8d97522)

Is there anything I can do to fix this?

**Comment by belom88** [comment]
I got same issue
